.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     - name: Install test dependencies
       run: sudo apt install  tcl8.6 tclx
 
-    - name: Build memKeydB
+    - name: Build TieredMemDB
```

Comment:
Please, merge this commit with the one that introduces memkind in github actions. No reason for keeping a fix commit with the commit introducing changes in the very same PR.